twake/frontend/src/app/services/channels/channels.js

```diff
@@ -50,12 +50,13 @@ class Channels extends Observable {
   readChannelIfNeeded(channel) {}
 
   async openDiscussion(membersIds, companyId = null) {
+    membersIds = membersIds.map(e => e); //Copy original list
```

Comment:
And the issue on ctrl+K was here, because you pass directly the channel members list in "handleElementType" in WorkspaceChannelList.tsx 
So when we do a membersIds.push(UsersService.getCurrentUserId()); we do it on the original members list from the original  channel resource.